packages/okta-vue/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 1.0.4
+
+### Bug fixes
+
+- [`95c3e62`](https://github.com/okta/okta-oidc-js/commit/dbfb7de3b41e932559ffa70790eeeca1dd30c270) - Fixes login_required error.
```

Comment:
Lets be a little more verbose here - as the fix satisfied a bigger problem. What do you think about:

> Fixes an issue where the library would enter an error state when attempting to renew expired tokens

Reply:
Makes sense but I would leave the name of the error (`login_required`), since many people can identify their problem easily thanks to that ^^